dkroutingtool/src/py/manual_viz.py

```diff
@@ -41,6 +43,15 @@ def write_manual_output(node_data, routes_for_mapping, vehicles, zone_route_map)
                 else:
                     node_num.append(node_i)
                     node_i += 1
+
+            #formulas
+            node_name.insert(formula_index,f'=COUNTIF(A:A,"{route_id}")-2')
```

Comment:
I find this a bit confusing. I know we are building up the rows of the dataframe, but these formulas don't really fit into these lists as is. I feel like it would be cleaner to apply these operations on the dataframe after it is created

Reply:
Yeah we could! I can break it out.